Softeq.XToolkit.Common.iOS/Extensions/NSStringExtensions.cs

```diff
@@ -79,6 +79,23 @@ public static NSMutableAttributedString BuildAttributedString(this string input)
             return self;
         }
 
+        public static NSMutableAttributedString LinksFont(
+            this NSMutableAttributedString self, UIFont font)
+        {
```

Comment:
Looks like we also should call `self.BeginEditing()` before and `self.EndEditing()` after changes:

- https://stackoverflow.com/questions/43723345/nsattributedstring-change-the-font-overall-but-keep-all-other-attributes
- https://developer.apple.com/documentation/foundation/nsmutableattributedstring/1411853-beginediting

Reply:
Fixed